orchagent/port.h

```diff
@@ -21,6 +21,8 @@ class Port
         PHY,
         MGMT,
         LOOPBACK,
+        DOT1D_BRIDGE,   // XXX: Not supported right now
+        DOT1Q_BRIDGE,
```

Comment:
BRIDGE_PORT?

---
In reply to: [126779862](https://github.com/Azure/sonic-swss/pull/250#discussion_r126779862) [](ancestors = 126779862)